packages/material-ui/src/ListItemText/ListItemText.js

```diff
@@ -5,109 +5,91 @@ import withStyles from '../styles/withStyles';
 import Typography from '../Typography';
 import ListContext from '../List/ListContext';
 
-export const styles = theme => ({
+export const styles = {
   /* Styles applied to the root element. */
   root: {
     flex: '1 1 auto',
     minWidth: 0,
-    padding: '0 16px',
-    '&:first-child': {
-      paddingLeft: 0,
-    },
+    marginTop: 4,
+    marginBottom: 4,
   },
+  /* Styles applied to the `Typography` components if primary and secondary are set. */
+  multilines: {
```

Comment:
multiline vs multilines?

Reply:
I think it's either `multiple lines` or `multiline`. `multilines` doesn't seem to be a valid english word (at least not in the oxford dictionary).